docs/syntax/origamijson.md

```diff
@@ -36,6 +36,10 @@ All origami components, whether modules or web services, should be discoverable
 	<td>integer*</td>
 	<td>Version of Origami to which the component conforms.  Currently must be set to 1.</td>
 </tr><tr>
+	<td><code>&nbsp;&nbsp;keywords</code></td>
+	<td>string*</td>
+	<td>Keywords related to the component to help discovery in the Registry. These can be stored as a comma separate string, i.e. "colours, palette, pink" for <code>o-colors</code>.</td>
```

Comment:
Yeh, probably right. It works regardless of how you store the keywords, but I guess comma-separated is a bit easier visually. Will update.